### PR TITLE
mockgcp support for servicenetworking

### PR DESCRIPTION
**Pull request by justinsb**
- mockgcp: generated proto & code for servicenetworking
- mockgcp: add support for servicenetworking connections

### CODE REVIEW

**Comment by justinsb** [comment]
WIP because needs ComputeAddress (at least) support for test fixtures - #1165

File: tests/e2e/unified_test.go [hunk 1]
```diff
@@ -453,6 +453,10 @@ func testFixturesInSeries(ctx context.Context, t *testing.T, testPause bool, can
 					addReplacement("metadata.createTime", "2024-04-01T12:34:56.123456Z")
 					addReplacement("metadata.genericMetadata.createTime", "2024-04-01T12:34:56.123456Z")
 
+					addReplacement("insertTime", "2024-04-01T12:34:56.123456Z")
```
Comment (anhdle-sso) [inline tests/e2e/unified_test.go:456]: These 2 normalizers are already added above (line 448-449)

Comment (justinsb) [reply tests/e2e/unified_test.go:456]: Good call - rebase issue!

**Comment by justinsb** [comment]
It's passing tests now @anhdle-sso 

/assign @anhdle-sso

**Pull request merged**